images/base/Dockerfile

```diff
@@ -55,6 +55,14 @@ ARG CNI_PLUGINS_AMD64_SHA256SUM="962100bbc4baeaaa5748cdbfce941f756b1531c2eadb290
 ARG CNI_PLUGINS_ARM64_SHA256SUM="ef17764ffd6cdcb16d76401bac1db6acc050c9b088f1be5efa0e094ea3b01df0"
 ARG CNI_PLUGINS_PPC64LE_SHA256SUM="5bd3c82ef248e5c6cc388f25545aa5a7d318778e5f9bc0a31475361bb27acefe"
 
+# Configure containerd-fuse-overlayfs snapshotter binary from upstream
+ARG CONTAINERD_FUSE_OVERLAYFS_VERSION="1.0.1"
+ARG CONTAINERD_FUSE_OVERLAYFS_TARBALL="v${CONTAINERD_FUSE_OVERLAYFS_VERSION}/containerd-fuse-overlayfs-${CONTAINERD_FUSE_OVERLAYFS_VERSION}-linux-${TARGETARCH}.tar.gz"
+ARG CONTAINERD_FUSE_OVERLAYFS_URL="https://github.com/AkihiroSuda/containerd-fuse-overlayfs/releases/download/${CONTAINERD_FUSE_OVERLAYFS_TARBALL}"
+ARG CONTAINERD_FUSE_OVERLAYFS_AMD64_SHA256SUM="ffeccbf101895193717f5cce8fa5197070af86f277783e22809f7d767ec5034f"
+ARG CONTAINERD_FUSE_OVERLAYFS_ARM64_SHA256SUM="f20e8a8fb812a0341eb8f63a3747efe7bdb4f2ee4be280e4e2a9d2fe42be5f8f"
+ARG CONTAINERD_FUSE_OVERLAYFS_PPC64LE_SHA256SUM="e6a67d58f5b31c8ae2ab7a67cb87992b40d91a5686b60ca5350cc3197f648b08"
+
```

Comment:
FYI these SHA256SUMS can be verified here https://github.com/AkihiroSuda/containerd-fuse-overlayfs/runs/2054687860?check_suite_focus=true

Reply:
Moved the repo: https://github.com/containerd/fuse-overlayfs-snapshotter/runs/2153593746?check_suite_focus=true#step:6:7